javascript/chronometer.js

```diff
@@ -1,26 +1,63 @@
 class Chronometer {
   constructor() {
-    // ... your code goes here
-  }
+  // ... your code goes here
+ 
+this.currentTime = 0;
+this.intervalId = 0;
+
+  }
+
+
   startClick(callback) {
     // ... your code goes here
+    setInterval(() => {
+      callback();
+      this.currentTime += 1;
+    },1000);
   }
   getMinutes() {
     // ... your code goes here
+
+let minutes = 0;
```

Comment:
check indentation of elements here, they should look inside the function for readability.

Reply:
okay. i will check it.